spec/dor_indexing/indexers/descriptive_metadata_indexer_contributor_spec.rb

```diff
@@ -12,7 +12,7 @@
     )
   end
 
-  describe 'primary contributor mappings from Cocina to Solr contributor_text_nostem_im and author_display_ss' do
+  describe 'primary contributor mappings from Cocina to Solr contributor_text_nostem_im, author_text_nostem_im and author_display_ss' do
```

Comment:
@ndushay What would you think about decoupling the RSpec description from the implementation details? Maybe something like `primary contributor mappings from Cocina to Solr` and leave it at that?